src/init.cpp

```diff
@@ -820,10 +811,13 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     LogPrintf("Using at most %i connections (%i file descriptors available)\n", nMaxConnections, nFD);
     std::ostringstream strErrors;
 
-    LogPrintf("Using %u threads for script verification\n", nScriptCheckThreads);
-    // BU: parallel block validation - begin
-    AddAllScriptCheckQueuesAndThreads(nScriptCheckThreads, &threadGroup); // This initializes and creates 4 separate script thread queues and thread pools.
-    // BU: parallel block validation - end
+    // -par=0 means autodetect, but passing 0 to the CParallelValidation constructor means no concurrency
+    int nPVThreads = GetArg("-par", DEFAULT_SCRIPTCHECK_THREADS);
+    if (nPVThreads <= 0)
+        nPVThreads += GetNumCores();
```

Comment:
i'm not seeing the rest of this code...there seems to be several lines missing

` if (nScriptCheckThreads <= 0)		
 -        nScriptCheckThreads += GetNumCores();		
 -    if (nScriptCheckThreads <= 1)		
 -        nScriptCheckThreads = 0;		
 -    else if (nScriptCheckThreads > MAX_SCRIPTCHECK_THREADS)		
 -        nScriptCheckThreads = MAX_SCRIPTCHECK_THREADS;		`
 -